packages/webapp/webapp_server.js

```diff
@@ -412,17 +415,32 @@ function getBoilerplateAsync(request, arch) {
   }));
 }
 
+// Notification hook whenever the runtime configuration is updated
+// hook will pass an object with the following fields:
+// - arch
+// - manifest
+// - runtimeConfig
+WebApp.addUpdatedNotifyHook = function(hook) {
```

Comment:
We have the `callback-hook` package which could have been used here.

Reply:
We still have some time before release, so that would be great! Just make sure that it works as you desired.

Reply:
done: https://github.com/meteor/meteor/pull/11615